posts/Ippon-USA-offers-a-helping-hand-to-those-impacted-by-natural-disasters.md

```diff
@@ -0,0 +1,32 @@
+My Time on Project:
+15 hours after setting off from my apartment in NYC and taking nearly every mode of transport available, I arrived at the HQ of the volunteer program in Oaxaca, Mexico. The so-called base is located in small neighbourhood La Santita, a barrio of Salinas Del Marquez. This town is separated by a massive harbour from the more well known Salina Cruz, famous for the oil refinery. 
+The next morning, after waking up early, the whole group of volunteers headed to the work site at 6.40 am. Currently, the group consist of approximately 40 people of all different ages from all over the world. All in purple, work boots on, sunscreen, water bottle and with a positive attitude: off to ‘work’. The volunteers make 8 long hour days, 6 days a week, in the extreme heat that marks this coastal area. Every now and then, they are blessed with a breeze. The region is known for high winds, but this year they are very absent, unfortunately. 
+On site, the group of volunteers works closely together with 15 masons, local labourers hired by the NGO. This comes with some difficulties from time to time because of the language barriers, just one of them speaks English and majority of the volunteers do not speak Spanish. Mostly, they work it out with sign language and the couple of words they know and it turns into a hilarious situation. Once a week, after work, the masons and volunteers will have ‘Spanglish’ classes, to learn the basics of the other language. 
+ 
+The people living in Salinas del Marquez are very generous, open and loving. The masons invite you to their houses, or take you to one of the local bars, in someone’s garden. I was, and still am, humbled by the generosity from people that live in totally different circumstances than what I am used to. I have met so many amazing people and have been lucky to hear their great stories. The kids are screaming for us when we encounter them on the streets, they are happy that we are there to rebuild their school. This connection with communities which we would otherwise miss in our daily lives is part of the reason i keep coming back to serve with this organisation. I caught the bug after joining them on project in Nepal 2016, initially a 4 week commitment which quickly escalated into a 5 month tenure on project to complete the school we were building there. I have since answered the call to serve on both international and domestic projects, both within the US and its territories, including Puerto Rico where I joined the cleanup following hurricane Maria. 
```

Comment:
```suggestion
The people living in Salinas del Marquez are very generous, open and loving. The masons invite you to their houses, or take you to one of the local bars, in someone’s garden. I was and still am humbled by the generosity of people that live in totally different circumstances than what I am used to. I have met so many amazing people and have been lucky to hear their great stories. The kids are screaming for us when we encounter them on the streets, and they are happy that we are there to rebuild their school. 

This connection with communities which we would otherwise miss in our daily lives is part of the reason I keep coming back to serve with this organization. I caught the bug after joining them on a project in Nepal 2016, initially a 4-week commitment which quickly escalated into a 5-month tenure on a project to complete the school we were building there. I have since answered the call to serve on both international and domestic projects, both within the US and its territories, including Puerto Rico where I joined the cleanup following hurricane Maria. 
```